content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -15,8 +15,7 @@ is a property of {{< glossary_tooltip text="Pods" term_id="pod" >}} that *attrac
 a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (either as a preference or a
 hard requirement). _Taints_ are the opposite -- they allow a node to repel a set of pods.
 
-_Tolerations_ are applied to pods, and allow (but do not require) the pods to schedule
-onto nodes with matching taints.
+_Tolerations_ are applied to pods, if it matches the Taints in any node it is scheduled, but it does not guarantee as scheduler has other paramters to decide.
```

Comment:
@dipankardas011 - please may I know what was the concern in adding the suggestion as-is [per the comment](https://github.com/kubernetes/website/pull/34067#discussion_r887682583)?

Reply:
Because @Huang-Wei was not convinced with the chages that why changed

Reply:
I'm not sure if you're reading the [same comment](https://github.com/kubernetes/website/pull/34067#discussion_r888181117) as I am, however @Huang-Wei please could you clarify which of the suggestions did you sign off on?